### PR TITLE
Adding support for refreshing web page [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- Adding refresh dropdown select field in Settings
- Using a `setInterval` in widget to refresh `<iframe>` by setting the `src` value again based on the value of the "refresh" param
- Added e2e test, set up integration test but couldn't get to work. Problems with requiring use of fake timers but also needing to use a timeout to wait until the frame has loaded. Need to revisit. 
- Fixing CCI build

### CODE REVIEW

**Comment by stulees** [comment]
@rodrigopavezi Can you review and I'll deploy right away once you have, thanks.

File: test/data/webpage-refresh.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(function(window) {
+  "use strict";
+
+  window.gadget = window.gadget || {};
+  window.gadget.settings = {
+    "params": {},
+    "additionalParams": {
+      "url": "http://www.risevision.com",
+      "refresh": 60000 // 1 minute
```
Comment (rodrigopavezi) [inline test/data/webpage-refresh.js]: @stulees I think it is missing region in here.

```
"region": {
        showRegion: "region",
        horizontal: 10,
        vertical: 20
      },
```

Comment (rodrigopavezi) [inline test/data/webpage-refresh.js]: @stulees ops, never mind. I didn't notice it is a new file.

**Comment by rodrigopavezi** [comment]
@stulees looks good :+1: cheers

**Comment by stulees** [comment]
@rodrigopavezi Good catch, have now added. Thanks

**Pull request merged**